graylog2-plugin-interfaces/src/main/java/org/graylog2/plugin/KafkaJournalConfiguration.java

```diff
@@ -32,6 +33,25 @@
 
 public class KafkaJournalConfiguration {
 
+    public KafkaJournalConfiguration() { }
+
+    @JsonCreator
+    public KafkaJournalConfiguration(@JsonProperty("directory") File messageJournalDir,
```

Comment:
Maybe add `@NotNull` here so that Jersey/JAX-RS can check the validity of the incoming data when deserializing it.